x-pack/plugins/siem/public/resolver/view/submenu.tsx

```diff
@@ -142,23 +150,35 @@ const NodeSubMenuComponents = React.memo(
      * When called with a set of `optionsWithActions`:
      * Render with a panel of options that appear when the menu host button is clicked
      */
+
+    const button = (
+      <EuiButton
+        onClick={
+          typeof optionsWithActions === 'object' ? handleMenuOpenClick : handleMenuActionClick
+        }
+        color={buttonColor}
+        size="s"
+        iconType={menuIsOpen ? 'arrowUp' : 'arrowDown'}
+        iconSide="right"
+        tabIndex={-1}
+      >
+        {menuTitle}
+      </EuiButton>
+    );
+
     return (
       <div className={className + (menuIsOpen ? ' is-open' : '')}>
-        <EuiButton
-          onClick={
-            typeof optionsWithActions === 'object' ? handleMenuOpenClick : handleMenuActionClick
-          }
-          color="ghost"
-          size="s"
-          iconType={menuIsOpen ? 'arrowUp' : 'arrowDown'}
-          iconSide="right"
-          tabIndex={-1}
+        <EuiPopover
```

Comment:
:red_circle:   This is interesting - might have to check this out later. If I read correctly, this will make the submenus _not_ show up by default? We may want to check that with @lindseypoli - some User Feedback was in the vein of wanting to make this _more_ discoverable (and it seems like maybe this would make it _less_?) Although this would potentially help/address _other_ user feedback that indicated they _don't_ want to see anything in the menu unless there are actual events to show. In that case, though, (and it's the reason I'm tagging this as :red_circle: , we may want to _also_ check that the `optionsWithActions > 0` to realize the benefit of that (i.e. Don't show this submenu unless there is something to actually show).

Reply:
Yea, @lindseypoli and I had conversations around this earlier today. The name popover is a little misleading here, it really just is about the style of the dropdown list that is displayed when you click on the button more than anything else. There are going to be some style and size changes here, but the plan is to move towards a state that allows these menus to be visible more often. The point about not showing them at all when there are no actions though  is intriguing and definitely makes sense. Next time I demo the changes to Lindsey, we'll take a look at this as well. Thanks Brent!